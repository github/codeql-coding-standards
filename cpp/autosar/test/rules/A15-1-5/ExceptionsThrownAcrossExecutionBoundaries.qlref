rules/A15-1-5/ExceptionsThrownAcrossExecutionBoundaries.ql