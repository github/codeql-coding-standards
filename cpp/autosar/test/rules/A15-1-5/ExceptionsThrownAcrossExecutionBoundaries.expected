| test-diff.cpp:4:5:4:5 | f | Function $@ an exception of type $@ and is $@ across execution boundaries. | test-diff.cpp:6:5:6:12 | throw ... | throws | file://:0:0:0:0 | int | int | test.cpp:11:5:11:5 | call to f | called |
| test.cpp:3:5:3:5 | g | Function $@ an exception of type $@ and is $@ across execution boundaries. | test.cpp:5:5:5:12 | throw ... | throws | file://:0:0:0:0 | int | int | test.cpp:20:5:20:5 | call to g | called |
